packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue

```diff
@@ -1,46 +1,115 @@
+        <ul :class="$style['c-breadcrumbs-list']">
+            <template v-for="({ name, url, routerLink }, index) in links">
+                <li
+                    v-if="index !== 0"
+                    :key="`${index}_link`"
+                    :class="$style['c-breadcrumbs-separator']">
+                    >
+                </li>
+                <li
+                    :key="`${index}_separator`"
+                    :class="$style['c-breadcrumbs-item']">
+                    <router-link
+                        v-if="routerLink"
+                        :to="url"
+                        :class="[
+                            $style['c-breadcrumbs-link'],
+                            linkActiveClass(index)
+                        ]">
+                        {{ name }}
+                    </router-link>
+                    <a
+                        v-else
+                        :href="url"
+                        :class="[
+                            $style['c-breadcrumbs-link'],
+                            linkActiveClass(index)
+                        ]">
+                        {{ name }}
+                    </a>
+                </li>
+            </template>
+        </ul>
     </div>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
-import tenantConfigs from '../tenants';
 
 export default {
     name: 'Breadcrumbs',
-    components: {},
     props: {
-        locale: {
-            type: String,
-            default: ''
+        links: {
+            type: Array,
+            default: () => []
         }
     },
-    data () {
-        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
-        const localeConfig = tenantConfigs[locale];
-
-        return {
-            copy: { ...localeConfig }
-        };
+    methods: {
+        linkActiveClass (index) {
+            return index === this.links.length - 1 ? this.$style['c-breadcrumbs-link--active'] : '';
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
 
+$breadcrumbs-text-colour: $white;
+$breadcrumbs-background-colour: rgba($black, 0.6);
+$breadcrumbs-border-radius: 16px;
+$breadcrumbs-not-active-font-weight: $font-weight-bold;
+$breadcrumbs-active-font-weight: $font-weight-base;
+
 .c-breadcrumbs {
     display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
+}
+.c-breadcrumbs-list {
+    list-style: none;
+    background-color: $breadcrumbs-background-colour;
+    border-radius: $breadcrumbs-border-radius;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items: center;
+}
+.c-breadcrumbs-item {
+    padding: spacing(x0.5) spacing(x2);
+    @include media('<narrowMid') {
+        padding: spacing(x0.5) spacing(x2) spacing(x0.5) spacing();
+    }
+}
+.c-breadcrumbs-item, .c-breadcrumbs-separator {
```

Comment:
NIT can you put these selectors on separate lines
```suggestion
.c-breadcrumbs-item,
.c-breadcrumbs-separator {
```